server/lib/logger/index.ts

```diff
@@ -0,0 +1,130 @@
+import { createLogger, format, transports } from 'winston';
+
+import { settings } from '../../../app/settings/server';
+
+const { combine, timestamp, label, printf, errors } = format;
+
+const oldCustomLevels = {
+	levels: {
+		error: 0,
+		deprecation: 1,
+		warn: 2,
+		success: 3,
+		info: 4,
+		debug: 5,
+
+		// custom
+	},
+	colors: {
+		error: 'red',
+		deprecation: 'magenta',
+		warn: 'magenta',
+		success: 'green',
+		info: 'blue',
+		debug: 'blue',
+	},
+};
+
+const logFormat = (info: Record<string, string>): string => `[${ info.timestamp }] - ${ info.level.toUpperCase() } - ${ info.label }${ info.section ? ` -> ${ info.section }` : '' } | ${ info.message }`;
```

Comment:
Maybe this change is a good time to log a JSON string instead of plain text? : 🙈 I think the cloud team would like being able to parse logs easily

Reply:
there is a discussion whether we should use JSON or plain text.. the idea is that our plain text would be parsed as well (having a well defined format).. so in this case we may support both.

Reply:
`pino` uses JSON by default.. I'll probably keep using JSON for now =)